spec/support/hanami_commands.rb

```diff
@@ -15,6 +15,7 @@ def rackup(args = {}, &blk)
       end
 
       def server(args = {}, &blk)
+        args[:port] ||= rand(2300..9999)
```

Comment:
Ooh, neat. I didn't know about `Prime` :)

I like this. Could be shortened to:  `Prime.each(9999).select{ |p| p > 2300}.sample`